config/analytics_blocklist.yml

```
@@ -222,6 +222,8 @@
   - response_body
   - created_at
   - updated_at
+   hesa_trn_submissions:
```

Looks like a typo here:

```suggestion
  :hesa_trn_submissions:
```